src/prefect/agent/docker/agent.py

```diff
@@ -518,20 +518,27 @@ def populate_env_vars(
         else:
             api = config.cloud.api
 
-        env = {}
+        env = {}  # type: ignore
```

Comment:
Why do we need to ignore the type?

Reply:
excellent question, I'll try to track down the underlying issue, but I believe this failed code check without the ignore